common4j/src/main/com/microsoft/identity/common/java/opentelemetry/SpanName.java

```diff
@@ -33,6 +33,8 @@ public enum SpanName {
     GetAccounts,
     RemoveAccount,
     WorkplaceJoin,
+    preInterruptInteractive,
+    preInterruptSilent,
```

Comment:
These names don't necessarily reflect their what they are being used for.

We should have better names

Reply:
@mohitc1 any suggestions?

Reply:
It would be good to add comment describing what it's used for.